kokkos.presets.json

```diff
@@ -0,0 +1,63 @@
+{
+    "version"   : 3,
+    "cmakeMinimumRequired" : {
+        "major" : 3,
+        "minor" : 22,
```

Comment:
Again 3.16 is the minimum.

Reply:
I removed cmakeMinimumRequired from the presets altogether :checkered_flag: